pkg/config/app_config.go

```diff
@@ -349,13 +349,34 @@ func findConfigFile(filename string) (exists bool, path string) {
 		return true, legacyConfigPath
 	}
 
+	// if on macOs, default to looking in ~/.config/lazygit first since that's
```

Comment:
I generally don't like lengthy comments, but thought that this was obtuse enough to warrant a description of why we have to implement this behavior instead of using xdg.